edit: added use cases for Orchestrator Generator extension (#1200)
mia-platform/documentation: docs/console/company-configuration/providers/extensions/orchestrator-generator/use-cases.md

1. **Serverless Computing Services:** another serverless fully managed resource that makes you focus solely on the logic you need without caring about the bare metal under the hood. Here you can find Aws Lambdas, Azure Functions or GCP Cloud Functions.

If you are deploying on one of these resources, you can simply build your own mapping layer that takes in the configuration of your Project and translates it into your favourite cloud resources.

### Using alternative Kubernetes Package Managers

While Kubernetes provides its own system for deploying applications, tools like Helm gained popularity due to their ability to manage deployments as reusable charts.  
If this is your case, the Orchestrator Generator extension can help you achieving this: you just need to create a mapper from the Project configuration into the structure and format expected by your package manager (in this case, the constructs required for Helm charts).

### Integrating with Existing Orchestrator Workflows

> Many organizations already adopted custom deployment tools, processes, and scripts tailored to their chosen orchestrator and their specific infrastructure requirements. Some other ones might even separate application development from the containerization management. Rather than forcing you to adopt Mia-Platform workflow, the Orchestrator Generator Extension can act as a bridge providing another level of flexibility.

Reviewer: ```suggestion
Many organizations already adopted custom deployment tools, processes and scripts tailored to their chosen orchestrator and their specific infrastructure requirements, also with the aim to separate application development from the containerization management. Rather than forcing you to adopt Mia-Platform workflow, the Orchestrator Generator Extension can act as a bridge providing another level of flexibility.
```
